integrationtests/server-integration/third-party-server/src/test/resources/arquillian.xml

```diff
@@ -42,6 +42,7 @@
             </property>
             -->
             <property name="javaVmArguments">
+                -Dinfinispan.server.integration.data-source=${infinispan.server.integration.data-source}
```

Comment:
This also fixes test failures. The DEV CI won't see as a failure because by default we run with WF